.gitignore

```diff
@@ -0,0 +1 @@
+cdp-deployment-templates-test
```

Comment:
Shouldn't this have the `-test` part removed?
Or perhaps an extra line added?

Reply:
no, as the unit test is currently working against that repo, and I don't think we should run it against the real one.

Reply:
Okay.